### PR TITLE
Modify explore in other templates for siibra api

### PR DESCRIPTION
**Pull request by fsdavid**
> can't we do that here?
> we can get centroids for mni152 and colin left/right, just not a big brain

@xgui3783 Then it should work with the current way. Could be great to get them.

### CODE REVIEW

File: src/atlasComponents/parcellationRegion/region.base.ts [hunk 1]
```diff
@@ -264,59 +264,77 @@ export class RenderViewOriginDatasetLabelPipe implements PipeTransform{
...
-    
+    // atlas is the single source of truth
+
     const otherTemplates = fetchedTemplates
-      .filter(({ fullId }) => getIdFromFullId(fullId) !== selectedTemplateId)
-      .filter(({ ['@id']: id, fullId }) => atlasTemplateSpacesIds.includes(id || fullId))
+      .filter(({ ['@id']: id }) => id !== regionOfInterest.context.template['@id'])
+      .filter(({ ['@id']: id }) => atlasTemplateSpacesIds.includes(id))
+      .filter(({ ['@id']: id }) => regionOfInterest.availableIn.map(ai => ai.id).includes(id))
+
```
Comment (xgui3783) [inline src/atlasComponents/parcellationRegion/region.base.ts:281]: can you pull these 3 into a single filter call?

**Review by xgui3783 — CHANGES_REQUESTED** [review]
Just a minor fix.

**Comment by xgui3783** [comment]
also, another note. 

you updated `e2e/protractor.conf.js`, even though no material change has been made. I know your IDE did formatting, but it clutters the commit, as well as the history. For a reviewer, it's bad form for them to scroll through tens if not hundred lines of formatting changes.

It is always a good idea to check what is being committed/added, instead of blindly doing `git add .` or `git commit -a`.

I recommend before `git add` or `git commit -a`, go through the updated file(s), and check for:

- if any secrets have been added by mistake (including tokens)
- remove any unused imports/console.log/debugger (I think linter can eventually help here)
- unstage/reset any changes to file that are unsubstantial

**Comment by fsdavid** [comment]
You are right, I understand. Usually, I always try not to have that formatting thing, but yesterday I missed it.

**Pull request merged**